packages/mos-gateway/package.json

```diff
@@ -65,6 +65,7 @@
 	],
 	"dependencies": {
 		"@sofie-automation/server-core-integration": "1.44.0-in-development",
+		"@sofie-automation/shared-lib": "link:../shared-lib",
```

Comment:
This should probably be an done with a version number lib server-core-integration.

There isnt technically much/any difference, but consistency